app/vmalert/config_test.go

```diff
@@ -20,6 +23,23 @@ func TestParseGood(t *testing.T) {
 	}
 }
 
+func TestParseGroupInterval(t *testing.T){
+	groups, err := Parse([]string{"testdata/dir/rules3-group-interval-5m-good.rules"}, true)
+	if err != nil {
+		t.Errorf("error parsing files %s", err)
+	}
+	err = errors.New("failed to parse group interval")
+	for _,group := range groups{
+		if strings.Contains(group.Name,"Without") {
+			if group.Interval != *evaluationInterval{
+				t.Error(err)
+			}
+		}else if group.Interval != 5 * time.Minute   {
+			t.Error(err)
```

Comment:
please make error message more specific for this case: `group %q expected to have interval %v; got %v`

Reply:
Fixed